2019-12-14 10:10:48 WARN  DBService:26 - Unable to obtain connection to database
java.sql.SQLException: No suitable driver found for jdbc:postgresql://localhost:5432/
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.revature.bankingapplication.DBService.connect(DBService.java:24)
	at com.revature.bankingapplication.DBService.addAccount(DBService.java:36)
	at com.revature.bankingapplication.Bank.addAccount(Bank.java:14)
	at com.revature.Menu.menuSelection(Menu.java:52)
	at com.revature.Driver.main(Driver.java:7)
2019-12-14 10:48:33 WARN  DBService:26 - Unable to obtain connection to database
java.sql.SQLException: No suitable driver found for jdbc:postgresql://localhost:5432/
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.revature.bankingapplication.DBService.connect(DBService.java:24)
	at com.revature.bankingapplication.DBService.addAccount(DBService.java:36)
	at com.revature.bankingapplication.Bank.addAccount(Bank.java:14)
	at com.revature.Menu.menuSelection(Menu.java:52)
	at com.revature.Driver.main(Driver.java:7)
2019-12-14 11:04:15 WARN  DBService:26 - Unable to obtain connection to database
java.sql.SQLException: No suitable driver found for jdbc:postgresql://localhost:5432/
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.revature.bankingapplication.DBService.connect(DBService.java:24)
	at com.revature.bankingapplication.DBService.addAccount(DBService.java:36)
	at com.revature.bankingapplication.Bank.addAccount(Bank.java:14)
	at com.revature.Menu.menuSelection(Menu.java:52)
	at com.revature.Driver.main(Driver.java:7)
2019-12-14 19:56:27 WARN  DBService:26 - Unable to obtain connection to database
java.sql.SQLException: No suitable driver found for jdbc:postgresql://localhost:5432/
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.revature.bankingapplication.DBService.connect(DBService.java:24)
	at com.revature.bankingapplication.DBService.addAccount(DBService.java:36)
	at com.revature.bankingapplication.Bank.addAccount(Bank.java:14)
	at com.revature.Actions.CustomerOptions.newCustomerActions(CustomerOptions.java:48)
	at com.revature.Menu.menuSelection(Menu.java:26)
	at com.revature.Driver.main(Driver.java:7)
2019-12-14 20:36:40 WARN  DBService:26 - Unable to obtain connection to database
java.sql.SQLException: No suitable driver found for jdbc:postgresql://localhost:5432/
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.revature.bankingapplication.DBService.connect(DBService.java:24)
	at com.revature.bankingapplication.DBService.addAccount(DBService.java:36)
	at com.revature.bankingapplication.Bank.addAccount(Bank.java:14)
	at com.revature.Actions.CustomerOptions.newCustomerActions(CustomerOptions.java:48)
	at com.revature.Menu.menuSelection(Menu.java:26)
	at com.revature.Driver.main(Driver.java:7)
2019-12-14 20:46:31 WARN  DBService:26 - Unable to obtain connection to database
java.sql.SQLException: No suitable driver found for jdbc:postgresql://localhost:5432/
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.revature.bankingapplication.DBService.connect(DBService.java:24)
	at com.revature.bankingapplication.DBService.addAccount(DBService.java:36)
	at com.revature.bankingapplication.Bank.addAccount(Bank.java:14)
	at com.revature.Actions.CustomerOptions.newCustomerActions(CustomerOptions.java:48)
	at com.revature.Menu.menuSelection(Menu.java:26)
	at com.revature.Driver.main(Driver.java:7)
2019-12-14 20:51:16 WARN  DBService:26 - Unable to obtain connection to database
java.sql.SQLException: No suitable driver found for jdbc:postgresql://localhost:5432/
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.revature.bankingapplication.DBService.connect(DBService.java:24)
	at com.revature.bankingapplication.DBService.addAccount(DBService.java:36)
	at com.revature.bankingapplication.Bank.addAccount(Bank.java:14)
	at com.revature.Actions.CustomerOptions.newCustomerActions(CustomerOptions.java:48)
	at com.revature.Menu.menuSelection(Menu.java:26)
	at com.revature.Driver.main(Driver.java:7)
2019-12-14 21:01:37 WARN  DBService:26 - Unable to obtain connection to database
java.sql.SQLException: No suitable driver found for jdbc:postgresql://localhost:5432/
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.revature.bankingapplication.DBService.connect(DBService.java:24)
	at com.revature.bankingapplication.DBService.addAccount(DBService.java:36)
	at com.revature.bankingapplication.Bank.addAccount(Bank.java:14)
	at com.revature.Actions.CustomerOptions.newCustomerActions(CustomerOptions.java:48)
	at com.revature.Menu.menuSelection(Menu.java:26)
	at com.revature.Driver.main(Driver.java:7)
2019-12-14 21:14:42 WARN  DBService:26 - Unable to obtain connection to database
java.sql.SQLException: No suitable driver found for jdbc:postgresql://localhost:5432/
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.revature.bankingapplication.DBService.connect(DBService.java:24)
	at com.revature.bankingapplication.DBService.addAccount(DBService.java:36)
	at com.revature.bankingapplication.Bank.addAccount(Bank.java:14)
	at com.revature.Actions.CustomerOptions.newCustomerActions(CustomerOptions.java:48)
	at com.revature.Menu.menuSelection(Menu.java:26)
	at com.revature.Driver.main(Driver.java:7)
2019-12-14 21:23:48 WARN  DBService:74 - Unable to add account
org.postgresql.util.PSQLException: ERROR: column "type" of relation "accounts" does not exist
  Position: 23
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2505)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2241)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:158)
	at org.postgresql.jdbc.PgPreparedStatement.executeUpdate(PgPreparedStatement.java:124)
	at com.revature.bankingapplication.DBService.addAccount(DBService.java:56)
	at com.revature.bankingapplication.Bank.addAccount(Bank.java:14)
	at com.revature.Actions.CustomerOptions.newCustomerActions(CustomerOptions.java:48)
	at com.revature.Menu.menuSelection(Menu.java:26)
	at com.revature.Driver.main(Driver.java:7)
2019-12-14 21:48:58 WARN  DBService:147 - Unable to update account
org.postgresql.util.PSQLException: ERROR: syntax error at or near "Users"
  Position: 8
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2505)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2241)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:158)
	at org.postgresql.jdbc.PgPreparedStatement.executeUpdate(PgPreparedStatement.java:124)
	at com.revature.bankingapplication.DBService.deleteAccount(DBService.java:144)
	at com.revature.bankingapplication.Bank.deleteAccount(Bank.java:26)
	at com.revature.Actions.UserOptions.userActions(UserOptions.java:52)
	at com.revature.Menu.menuSelection(Menu.java:30)
	at com.revature.Driver.main(Driver.java:7)
2019-12-16 12:44:58 WARN  DBService:176 - Unable to update account
org.postgresql.util.PSQLException: ERROR: syntax error at or near "Users"
  Position: 8
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2505)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2241)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:158)
	at org.postgresql.jdbc.PgPreparedStatement.executeUpdate(PgPreparedStatement.java:124)
	at com.revature.bankingapplication.DBService.deleteAccount(DBService.java:173)
	at com.revature.bankingapplication.Bank.deleteAccount(Bank.java:27)
	at com.revature.Actions.UserOptions.userActions(UserOptions.java:59)
	at com.revature.Menu.menuSelection(Menu.java:30)
	at com.revature.Driver.main(Driver.java:7)
2019-12-16 14:55:24 WARN  DBService:26 - Unable to obtain connection to database
org.postgresql.util.PSQLException: FATAL: sorry, too many clients already
	at org.postgresql.core.v3.ConnectionFactoryImpl.doAuthentication(ConnectionFactoryImpl.java:520)
	at org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:141)
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:192)
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:211)
	at org.postgresql.Driver.makeConnection(Driver.java:458)
	at org.postgresql.Driver.connect(Driver.java:260)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.revature.bankingapplication.DBService.connect(DBService.java:24)
	at com.revature.bankingapplication.DBService.updateStatus(DBService.java:131)
	at com.revature.bankingapplication.Bank.updateStatus(Bank.java:77)
	at com.revature.Actions.EmployeeOptions.employeeActions(EmployeeOptions.java:48)
	at com.revature.Menu.menuSelection(Menu.java:35)
	at com.revature.Driver.main(Driver.java:7)
2019-12-16 14:56:18 WARN  DBService:26 - Unable to obtain connection to database
org.postgresql.util.PSQLException: FATAL: sorry, too many clients already
	at org.postgresql.core.v3.ConnectionFactoryImpl.doAuthentication(ConnectionFactoryImpl.java:520)
	at org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:141)
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:192)
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:211)
	at org.postgresql.Driver.makeConnection(Driver.java:458)
	at org.postgresql.Driver.connect(Driver.java:260)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.revature.bankingapplication.DBService.connect(DBService.java:24)
	at com.revature.bankingapplication.DBService.getAccount(DBService.java:83)
	at com.revature.bankingapplication.Bank.getAccount(Bank.java:18)
	at com.revature.Actions.UserOptions.userActions(UserOptions.java:43)
	at com.revature.Menu.menuSelection(Menu.java:31)
	at com.revature.Driver.main(Driver.java:7)
2019-12-16 16:29:02 WARN  DBService:26 - Unable to obtain connection to database
org.postgresql.util.PSQLException: FATAL: sorry, too many clients already
	at org.postgresql.core.v3.ConnectionFactoryImpl.doAuthentication(ConnectionFactoryImpl.java:520)
	at org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:141)
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:192)
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:211)
	at org.postgresql.Driver.makeConnection(Driver.java:458)
	at org.postgresql.Driver.connect(Driver.java:260)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.revature.bankingapplication.DBService.connect(DBService.java:24)
	at com.revature.bankingapplication.DBService.getAccount(DBService.java:86)
	at com.revature.bankingapplication.Bank.getAccount(Bank.java:18)
	at com.revature.Actions.UserOptions.userActions(UserOptions.java:31)
	at com.revature.Menu.menuSelection(Menu.java:31)
	at com.revature.Driver.main(Driver.java:7)
2019-12-16 16:46:43 WARN  DBService:26 - Unable to obtain connection to database
org.postgresql.util.PSQLException: FATAL: sorry, too many clients already
	at org.postgresql.core.v3.ConnectionFactoryImpl.doAuthentication(ConnectionFactoryImpl.java:520)
	at org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:141)
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:192)
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:211)
	at org.postgresql.Driver.makeConnection(Driver.java:458)
	at org.postgresql.Driver.connect(Driver.java:260)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.revature.bankingapplication.DBService.connect(DBService.java:24)
	at com.revature.bankingapplication.DBService.getAccount(DBService.java:86)
	at com.revature.bankingapplication.Bank.getAccount(Bank.java:18)
	at com.revature.Actions.UserOptions.userActions(UserOptions.java:31)
	at com.revature.Menu.menuSelection(Menu.java:31)
	at com.revature.Driver.main(Driver.java:7)
2019-12-16 17:04:37 WARN  DBService:26 - Unable to obtain connection to database
org.postgresql.util.PSQLException: FATAL: sorry, too many clients already
	at org.postgresql.core.v3.ConnectionFactoryImpl.doAuthentication(ConnectionFactoryImpl.java:520)
	at org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:141)
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:192)
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:211)
	at org.postgresql.Driver.makeConnection(Driver.java:458)
	at org.postgresql.Driver.connect(Driver.java:260)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.revature.bankingapplication.DBService.connect(DBService.java:24)
	at com.revature.bankingapplication.DBService.getAccount(DBService.java:86)
	at com.revature.bankingapplication.Bank.getAccount(Bank.java:18)
	at com.revature.Actions.UserOptions.userActions(UserOptions.java:31)
	at com.revature.Menu.menuSelection(Menu.java:31)
	at com.revature.Driver.main(Driver.java:7)
2019-12-17 13:36:35 INFO  UserOptions:33 - Succesful log in attempt
2019-12-17 13:36:41 INFO  UserOptions:50 - Account 7 retrieved succesfully
2019-12-17 13:57:43 INFO  EmployeeOptions:27 - Succesful employee account verification
2019-12-17 13:57:55 WARN  DBService:212 - Unable to retrieve all accounts
2019-12-17 13:57:55 INFO  EmployeeOptions:42 - Succesfully retrieved all accounts
2019-12-17 14:00:22 INFO  CustomerOptions:55 - Succesfully created new account
2019-12-17 14:01:03 INFO  UserOptions:119 - Failed user account verification
2019-12-17 14:01:16 INFO  UserOptions:33 - Succesful user account verification
2019-12-17 14:01:47 INFO  UserOptions:50 - Account 8 retrieved succesfully
2019-12-17 14:02:07 INFO  UserOptions:33 - Succesful user account verification
2019-12-17 14:02:17 INFO  UserOptions:58 - Succesfully deposited 1000 into account 8
2019-12-17 14:02:59 INFO  UserOptions:119 - Failed user account verification
2019-12-17 14:03:27 INFO  UserOptions:33 - Succesful user account verification
2019-12-17 14:49:33 INFO  UserOptions:33 - Succesful user account verification
2019-12-17 15:12:36 INFO  CustomerOptions:62 - Succesfully created new account
2019-12-17 15:18:02 INFO  CustomerOptions:58 - Succesfully created new account
2019-12-17 15:27:30 INFO  UserOptions:120 - Failed user account verification
2019-12-17 15:28:30 INFO  UserOptions:120 - Failed user account verification
2019-12-17 15:35:43 INFO  UserOptions:120 - Failed user account verification
2019-12-17 15:44:29 INFO  EmployeeOptions:82 - Failed account verification
2019-12-17 15:44:37 INFO  EmployeeOptions:28 - Succesful employee account verification
2019-12-17 18:39:25 WARN  Menu:53 - Invalid input during menu selection
2019-12-17 18:39:57 WARN  Menu:53 - Invalid input during menu selection
2019-12-17 18:40:31 INFO  CustomerOptions:57 - Succesfully created new account
2019-12-17 18:42:15 INFO  CustomerOptions:57 - Succesfully created new account
2019-12-17 18:43:33 INFO  UserOptions:120 - Failed user account verification
2019-12-17 18:46:37 INFO  UserOptions:34 - Succesful user account verification
2019-12-17 18:46:40 INFO  UserOptions:51 - Account 11 retrieved succesfully
2019-12-17 18:46:59 INFO  UserOptions:34 - Succesful user account verification
2019-12-17 18:47:06 INFO  UserOptions:59 - Succesfully deposited 1000 into account 11
2019-12-17 18:47:22 INFO  UserOptions:34 - Succesful user account verification
2019-12-17 18:47:34 INFO  UserOptions:72 - Succesfully withdrew 495 from account 11
2019-12-17 18:48:05 INFO  UserOptions:34 - Succesful user account verification
2019-12-17 18:48:25 INFO  UserOptions:94 - Succesfully withdrew 100 from account 11
2019-12-17 18:48:25 INFO  UserOptions:104 - Succesfully deposited 100 into account 11
2019-12-17 18:49:52 INFO  UserOptions:120 - Failed user account verification
2019-12-17 18:50:02 INFO  UserOptions:34 - Succesful user account verification
2019-12-17 18:50:07 INFO  UserOptions:82 - Succesfully deleted account 11
2019-12-17 18:50:32 INFO  EmployeeOptions:28 - Succesful employee account verification
2019-12-17 18:50:37 WARN  DBService:212 - Unable to retrieve all accounts
2019-12-17 18:55:06 INFO  EmployeeOptions:28 - Succesful employee account verification
2019-12-17 18:55:09 WARN  DBService:212 - Unable to retrieve all accounts
2019-12-17 18:55:30 WARN  DBService:212 - Unable to retrieve all accounts
2019-12-17 19:04:17 WARN  DBService:212 - Unable to retrieve all accounts
2019-12-17 19:05:19 WARN  Menu:53 - Invalid input during menu selection
2019-12-17 19:05:38 WARN  DBService:212 - Unable to retrieve all accounts
2019-12-17 19:13:14 WARN  DBService:211 - Unable to retrieve all accounts
2019-12-17 19:14:05 WARN  DBService:211 - Unable to retrieve all accounts
2019-12-17 19:28:13 INFO  EmployeeOptions:28 - Succesful employee account verification
2019-12-17 19:28:16 WARN  DBService:211 - Failed to retrieve all accounts
2019-12-17 19:28:27 INFO  EmployeeOptions:28 - Succesful employee account verification
2019-12-17 19:28:30 INFO  EmployeeOptions:49 - Succesfully viewed account 12
2019-12-17 19:42:37 WARN  DBService:110 - Unable to retrieve account
java.lang.Exception: Unknown account type
	at com.revature.bankingapplication.DBService.getAccount(DBService.java:101)
	at com.revature.bankingapplication.Bank.getAccount(Bank.java:24)
	at com.revature.bankingapplication.Bank.updateStatus(Bank.java:83)
	at com.revature.Actions.Administration.administrativeActions(Administration.java:90)
	at com.revature.Menu.menuSelection(Menu.java:43)
	at com.revature.Driver.main(Driver.java:7)
2019-12-17 19:46:11 INFO  EmployeeOptions:28 - Succesful employee account verification
2019-12-17 19:48:24 INFO  EmployeeOptions:28 - Succesful employee account verification
2019-12-17 19:48:36 INFO  EmployeeOptions:49 - Succesfully viewed account 10
2019-12-17 19:48:56 INFO  EmployeeOptions:28 - Succesful employee account verification
2019-12-17 19:49:55 INFO  EmployeeOptions:28 - Succesful employee account verification
2019-12-17 20:07:16 INFO  Administration:46 - Succesfully retrieved all accounts
2019-12-17 20:07:59 INFO  Administration:63 - Succesfully deposited 1000 into account 9
2019-12-17 20:08:29 INFO  Administration:78 - Succesfully withdrew 100 from account 9
2019-12-17 20:09:10 INFO  Administration:102 - Acccount 12 denied approval 
2019-12-18 09:47:31 WARN  DBService:106 - No user account was found for ID10
2019-12-18 09:52:37 WARN  DBService:106 - No user account was found for ID10
2019-12-18 09:58:04 WARN  DBService:106 - No user account was found for ID10
2019-12-18 10:02:25 WARN  DBService:106 - No user account was found for ID10
2019-12-18 10:02:51 WARN  DBService:106 - No user account was found for ID10
2019-12-18 10:06:11 WARN  DBService:106 - No user account was found for ID10
2019-12-18 10:06:11 WARN  DBService:106 - No user account was found for ID11
2019-12-18 10:10:38 WARN  DBService:106 - No user account was found for ID10
2019-12-18 10:10:38 WARN  DBService:106 - No user account was found for ID11
2019-12-18 10:14:25 INFO  Administration:46 - Succesfully retrieved all accounts
2019-12-18 10:14:52 INFO  CustomerOptions:57 - Succesfully created new account
2019-12-18 10:15:00 INFO  EmployeeOptions:28 - Succesful employee account verification
2019-12-18 10:16:14 WARN  UserOptions:120 - Failed user account verification
2019-12-18 10:16:33 WARN  UserOptions:120 - Failed user account verification
2019-12-18 10:16:51 WARN  UserOptions:120 - Failed user account verification
2019-12-18 10:30:08 WARN  DBService:106 - No user account was found for ID11
